internal/record/recording_https_proxy.go

```diff
@@ -178,17 +161,40 @@ func (r *RecordingHTTPSProxy) proxyRequest(w http.ResponseWriter, req *http.Requ
 	return resp, respBodyBytes, nil
 }
 
-func (r *RecordingHTTPSProxy) recordResponse(resp *http.Response, fileName string, body []byte) error {
+func (r *RecordingHTTPSProxy) recordResponse(recReq *store.RecordedRequest, resp *http.Response, fileName string, body []byte) error {
 	recordedResponse, err := store.NewRecordedResponse(resp, body)
 	if err != nil {
 		return err
 	}
+	recordPath := filepath.Join(r.recordingDir, fileName+".http.log")
+	shaSum := recReq.ComputeSum()
 
-	recordedResponse.Body = r.redactor.Bytes(recordedResponse.Body)
+	// Open the file in append mode, create it if it doesn't exist.
+	file, err := os.OpenFile(recordPath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
If the Test-Name header isn't set this still is still content hash right? in that case we may have cross-test collisions? previously it wasn't a problem because same hash meant same request (, same previous requests, ) and same response, but now that we're appending it might be a problem?

Reply:
introduced seenMap to reset the recording when first time seen a request for a given test